themes/bootstrap3/templates/admin/feedback/home.phtml

```diff
@@ -175,4 +175,4 @@ $(document).ready(function() {
 });
 JS;
 ?>
-<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $js, 'SET');?>
+<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $js, 'SET');?>
```

Comment:
Looks like the end-of-file line break was removed here. For style reasons, we try to keep a line break on the last line of every file.